munet/mutest/userapi.py

```diff
@@ -476,7 +493,27 @@ def _match_command_json(
                 "JSON load failed. Check match value is in JSON format: %s", error
             )
 
-        if json_diff := json_cmp(expect, js):
+        if exact_match:
+            deep_diff = json_cmp(expect, js)
+            # Convert DeepDiff completely into dicts or lists at all levels
+            json_diff = json.loads(deep_diff.to_json())
+        else:
+            deep_diff = json_cmp(expect, js, ignore_order=True)
+            # Convert DeepDiff completely into dicts or lists at all levels
+            json_diff = json.loads(deep_diff.to_json())
+            # Remove new fields in json object from diff
+            if json_diff.get('dictionary_item_added') is not None:
+                del json_diff['dictionary_item_added']
```

Comment:
Does `json_cmp` always return these key-valus at the top level? i.e., what if you have a diff in a dict in a dict?
```
json1 = '{ "foo": { "bar": { "one": 1 } } }'
json2 = '{ "foo": { "bar": { "one": 1, "two": 2 } } }'
```

Reply:
ok thanks